openmls/src/framing/validation.rs

```diff
@@ -145,15 +180,15 @@ impl DecryptedMessage {
 #[derive(Debug)]
 pub struct UnverifiedMessage {
     plaintext: VerifiableMlsPlaintext,
-    credential: Option<Credential>,
+    credential: Credential,
     aad_option: Option<Vec<u8>>,
 }
 
 impl UnverifiedMessage {
     /// Construct an [UnverifiedMessage] from a [DecryptedMessage] and an optional [Credential].
     pub(crate) fn from_decrypted_message(
         decrypted_message: DecryptedMessage,
-        credential: Option<Credential>,
+        credential: Credential,
```

Comment:
This was optional because external (preconfigured) messaged don't have a credential. This shouldn't change.

Reply:
Good point. I'll reverse the change.